languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,19 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rule id="IL_FAUT" name="Il faut" default="temp_off">
+          <antipattern>
+              <token postag="M fin inte"/>
+          </antipattern>
+          <pattern>
+              <token postag="SENT_START"/>
+              <marker>
+                  <token>faut</token>
+              </marker>
+          </pattern>
+          <message>C'est une expression est familière. Dans le langage formel, il convient d'écrire <suggestion>Il <match no="2" case_conversion="startlower"/></suggestion>.</message>
```

Comment:
Typo: _C'est une expression **est** familière._

Maybe change to: "Cette expression est considérée comme familière. ..."